docs/Reference/troubleshooting.md

```diff
@@ -1,32 +1,108 @@
 
-| Error Issue or Message | Possible Solutions |
-| ------------- | ------------------ |
-| Intel&reg; AMT device fails to reconnect after MPS is down for 2 or more days | Update to latest firmware.  Workarounds include: a) Reboot device b) unplug and re-plug network cable |
+## Commonly Asked Questions
 
-# MPS
+**How do releases work with Open AMT?**
 
-| Error Issue or Message | Possible Solutions |
-| ------------- | ------------------ |
-| Vault is empty in dev mode. | Profiles and configs are not persistent in this mode. To run vault in production mode, follow the [Use Docker and Vault in Production Mode](../Reference/productionVault.md).|
-|MPS is missing from list of running services. | (1) Check for error messages in the logs. |
-| | (2) Verify that the .env file contains correct values in each field.|
+Open AMT carries two releases, the Rapid ("Monthly") Release and the Long-Term Support (LTS) Release. 
 
-# RPS
+Rapid Releases occur every 4-6 weeks. Support for security and bug fixes is only for the duration of that rapid release. Customers will be encouraged to move to a latest rapid release for the most up to date security and bug fixes.
 
-| Error Issue or Message | Possible Solutions |
-| ------------- | ------------------ |
-| Create a profile fails or information cannot be read from vault. | Make sure Vault and PostGres are running. For details, see the `docker ps` command in [Build and Run Docker Images](../GetStarted/setup.md#Builddockerimages).|
-| An error occurred during provisioning. | (1) Verify that the correct certificate is being used. |
-|  | (2) Verify the Domain suffix. |
+LTS Releases occur roughly every 1 to 1.5 years. Support for security and bug fixes exist until the next LTS version is available. At that point, we will provide migration documentation and support to help customers move over to the new LTS release.
+
+<br>
+
+**How does versioning work with Open AMT?**
+
+Open AMT follows [SemVer](https://semver.org/) practices for versioning. This means:
+
+- Major Version Increment - Breaking Changes (ex: 2.0.0 -> 3.0.0)
+- Minor Version Increment - New Features (ex: 2.0.0 -> 2.1.0)
+- Patch Version Increment - Security and Bug Fixes (ex: 2.0.0 -> 2.0.1)
+
+All microservices with the same minor version should be compatible.
+
+The separate repos for microservices and libraries are versioned individually. These versions are separate from the `open-amt-cloud-toolkit` repo version.  The `open-amt-cloud-toolkit` repo is where we have the monthly release. This repo might carry a higher version than some of the individual repos but is tagged as `{Month} {Year}`. All sub-repos referenced within `open-amt-cloud-toolkit` for a specific release are guaranteed to be compatible.
+
+<br>
+
+**What versions of Intel&reg; AMT are supported?**
+
+Open AMT aligns to the Intel NEX support roadmap for Intel vPro&reg; Platform and Intel&reg; AMT devices. This is currently calculated as `Latest AMT Version - 7`.
```

Comment:
Use Network and Edge group (NEX) may be?

Reply:
Updated!